helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -63,7 +63,7 @@ recommender:
   resources:
     limits:
       cpu: 200m
-      memory: 1000Mi
+      memory: 3000Mi
```

Comment:
Do you have a basis for those numbers?

Reply:
Yes, I based myself on the comments made by @TheoBrigitte on the related incident : 
> pod keep being killed due to OOM it seems
raising memory from 2G to 3G

Reply:
Alright :)